peachjam/js/components/FindDocuments/index.vue

```diff
@@ -66,8 +66,25 @@
         <div class="col-md-12 col-lg-9 search-pane position-relative">
           <div class="search-results">
             <div v-if="searchInfo.count">
-              <div class="mb-3 d-flex justify-content-between">
+              <div class="mb-3 sort-body">
                 <div>{{ searchInfo.count }} documents found.</div>
+                <div class="sort__inner">
+                  Sort by
+                  <select
+                    v-model="ordering"
+                    class="ms-2"
+                  >
+                    <option value="-score">
+                      Revelance
```

Comment:
Relevance

Reply:
covered in this pr https://github.com/laws-africa/peachjam/pull/535